src/test/java/de/retest/recheck/util/ReflectionUtilitiesTest.java

```diff
@@ -202,7 +202,7 @@ public void getSimpleName_should_return_classname() throws Exception {
 		assertThat( getSimpleName( "de.retest.Main$1" ) ).isEqualTo( "Main$1" );
 	}
 
-	private class A {};
+	private static class A {};
 
 	private class B extends A {};
```

Comment:
Sollte die Klasse B dann nicht auch static sein?

Reply:
Yes, it's just slipped into another commit: https://github.com/retest/recheck/commit/a3a433132da228449c8bc1dcbb7ca0eeca0373c9#diff-ebad5a247e6ad34c4d38ff43dbb9bf0536d7dd906ee2532746deb38ed4e8cd96L207